specs/adapters/mysql2_adapter_spec.rb

```diff
@@ -3,8 +3,9 @@
 describe "Spectacles::SchemaStatements::Mysql2Adapter" do
   config = {
     :adapter => "mysql2",
-    :host => "localhost",
-    :username => "root"
+    :host => ENV["MYSQL_HOST"] || "localhost",
```

Comment:
Presumably this makes it easier to run specs against specific databases and/or in Docker images?